codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/rustlang/CargoDependency.kt

```diff
@@ -181,6 +180,7 @@ data class CargoDependency(
     val features: Set<String> = emptySet(),
     val defaultFeatures: Boolean = true,
     val rustName: String = name.replace("-", "_"),
+    val `package`: String? = null,
```

Comment:
"package" is a reserved word in Kotlin but that's what the key is called in `Cargo.toml`s.